protecode/client.py

```diff
@@ -336,6 +359,10 @@ def add_triage_raw(
     # --- "rest" routes (undocumented API)
 
     def login(self):
+
+        if self._credentials.auth_scheme is not ProtecodeAuthScheme.BASIC_AUTH:
```

Comment:
nit: please check for all auth-types + add notimplemented in case we encounted a yet unknown one